Sources/xcproj/XCScheme.swift

```diff
@@ -169,9 +169,75 @@ final public class XCScheme {
         }
     }
 
+    final public class ExecutionAction {
+        public var title: String
+        public var scriptText: String
+        public var environmentBuildable: BuildableReference?
+
+        public init(scriptText: String, title: String? = nil, environmentBuildable: BuildableReference? = nil) {
+            self.scriptText = scriptText
+            self.title = title ?? "Run Script"
```

Comment:
What do you think, if you moved the default value to init arguments? This way it is more clear that title is not optional. At the moment, looking at the init method signature, it looks like title is optional.